Kitodo/src/main/webapp/WEB-INF/resources/css/pattern-library.css

```diff
@@ -745,7 +749,19 @@ button.ui-datepicker-trigger.ui-button-icon-only::before {
 .ui-treetable tbody td,
 .ui-treetable tfoot td {
     border: 0;
-    padding: 8px 16px;
+    padding: 0 0 0 16px;
+    line-height: 36px;
```

Comment:
Shoulnd't `line-height` come before `padding`, supposed the CSS properties are ordered alphabetically?